### PR TITLE
Added policy to deny resources in specified window.

### PR DESCRIPTION
**Pull request by anuddeeph1**
## Description
This policy denies the creation and updating of resources specifically for Deployment and Pod kinds during a specified time window. The policy is designed to enhance control over resource modifications during critical periods, ensuring stability and consistency within the Kubernetes environment.

## Checklist
- [] I have read the [policy contribution guidelines](https://github.com/kyverno/policies/blob/main/README.md#contribution).
- [] I have added test manifests and resources covering both positive and negative tests that prove this policy works as intended.
- [] I have added the artifacthub-pkg.yml file and have verified it is complete and correct.

### CODE REVIEW

**Review by chipzoller — CHANGES_REQUESTED** [review]
Neat concept. I think it can be useful. I note that your policy does not use the current modern schema. Can you please update this to conform to all the latest specifications?

**Comment by anuddeeph1** [comment]
@chipzoller , updated with new schema

File: other/deny-creation-of-resources/deny-creation-of-resources.yaml [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+  validationFailureAction: Audit
+  background: false
+  rules:
+  - name: deny-creation-of-resources
+    match:
+      any:
+      - resources:
+          kinds:
+            - Deployment
+            - Pod
```
Comment (chipzoller) [inline other/deny-creation-of-resources/deny-creation-of-resources.yaml:28]: Suggest only matching on Deployment and not Pod. Will need to update relevant areas of this policy and tests if so.

File: other/deny-creation-of-resources/deny-creation-of-resources.yaml [hunk 2]
```diff
@@ -0,0 +1,44 @@
...
+    preconditions:
+      all:
+      - key: '{{ time_now_utc().time_to_cron(@).split(@,'' '') | [1].to_number(@)
+          }}'
```
Comment (chipzoller) [inline other/deny-creation-of-resources/deny-creation-of-resources.yaml:32]: Should not be on two lines. Please also add a comment what this does.

File: other/deny-creation-of-resources/deny-creation-of-resources.yaml [hunk 3]
```diff
@@ -0,0 +1,44 @@
...
+        operator: AnyIn
+        value: 8-10
+    validate:
+      message: Creating and updating resources is not allowed at this time, please contact admin!
+      deny:
+        conditions:
+          all:
+          - key: '{{request.operation}}'
+            operator: In
```
Comment (chipzoller) [inline other/deny-creation-of-resources/deny-creation-of-resources.yaml:41]: `In` is deprecated and no longer available. Please use `AnyIn`.

File: other/deny-creation-of-resources/deny-creation-of-resources.yaml [hunk 4]
```diff
@@ -0,0 +1,44 @@
...
+            - Deployment
+            - Pod
+    preconditions:
+      all:
+      - key: '{{ time_now_utc().time_to_cron(@).split(@,'' '') | [1].to_number(@)
+          }}'
+        operator: AnyIn
+        value: 8-10
+    validate:
+      message: Creating and updating resources is not allowed at this time, please contact admin!
```
Comment (chipzoller) [inline other/deny-creation-of-resources/deny-creation-of-resources.yaml:36]:
```suggestion
      message: Creating and updating resources is not allowed at this time.
```

File: other/deny-creation-of-resources/deny-creation-of-resources.yaml [hunk 5]
```diff
@@ -0,0 +1,44 @@
+apiVersion: kyverno.io/v1
+kind: ClusterPolicy
+metadata:
+  name: deny-creation-of-resources
```
Comment (chipzoller) [inline other/deny-creation-of-resources/deny-creation-of-resources.yaml:4]: Suggest a different name. This isn't just about denying creation but also updates as well.

File: other/deny-creation-of-resources/deny-creation-of-resources.yaml [hunk 6]
```diff
@@ -0,0 +1,44 @@
...
+  annotations:
+    policies.kyverno.io/title: Deny Creation of Resources
```
Comment (chipzoller) [inline other/deny-creation-of-resources/deny-creation-of-resources.yaml:6]: Name

File: other/deny-creation-of-resources/deny-creation-of-resources.yaml [hunk 7]
```diff
@@ -0,0 +1,44 @@
...
+    policies.kyverno.io/description: >-
+      This policy denies the creation and updating of resources specifically for Deployment 
+      and Pod kinds during a specified time window. The policy is designed to enhance control 
+      over resource modifications during critical periods, ensuring stability and consistency 
+      within the Kubernetes environment.
+spec:
+  validationFailureAction: Audit
+  background: false
+  rules:
+  - name: deny-creation-of-resources
```
Comment (chipzoller) [inline other/deny-creation-of-resources/deny-creation-of-resources.yaml:22]: Rules should be named differently from policies.

**Review by chipzoller — CHANGES_REQUESTED** [review]
When making policy changes, you will need to modify corresponding/supporting files as well.

**Review by chipzoller — CHANGES_REQUESTED** [review]
Please replace instances of the word "updation", which only exists in Indian English, to "updating".

**Review by chipzoller — APPROVED** [review]
Thanks for seeing this process through, @anuddeeph1. You've done good work and we encourage you to make a couple other contributions to become a [recognized contributor](https://kyverno.io/community/#project-roles).

**Review by chipzoller — CHANGES_REQUESTED** [review]
Sorry, but Chainsaw tests are failing. Need to investigate and solve.

**Comment by anuddeeph1** [comment]
> Sorry, but Chainsaw tests are failing. Need to investigate and solve.

Resolved the issues in chainsaw test

**Review by chipzoller — APPROVED** [review]

**Pull request merged**